lib/line-protocol.js

```diff
@@ -169,13 +170,8 @@ module.exports.format = (event, config) => {
         return;
     }
 
-    const tags = internals.serialize(internals.tags(event));
+    const tags = internals.serialize(internals.tags(event, config));
     const eventValues = getEventValues(event);
-    if (config.metadata) {
-        Object.keys(config.metadata).forEach( (key) => {
```

Comment:
This breaks backwards compatibility for anyone passing in metadata other than `serviceName`.  Other metadata will be ignored.  Maybe keep this logic but add the results to the tag set instead of the field set?